src/Symfony/Component/Routing/Loader/AnnotationClassLoader.php

```diff
@@ -71,23 +52,55 @@
  */
 abstract class AnnotationClassLoader implements LoaderInterface
 {
+    /**
+     * @var Reader|null
+     *
+     * @deprecated in Symfony 6.4, this property will be removed in Symfony 7.
+     */
     protected $reader;
+
+    /**
+     * @var string|null
+     *
+     * @internal since Symfony 6.4, this property will be private in Symfony 7.
```

Comment:
I'll revert it since making the properties private is an unrelated change. However, in addition to adding a type here, I felt that these properties are in fact internal state of the `AnnotationClassLoader` class that a child class should not need to fiddle with.